README.md

```diff
@@ -4,7 +4,7 @@
 This is an auto TFT bot, with some decent logic built in.
 
 > [!NOTE]

```

Comment:
Would you mind adding a note (just since this PR is already updating a bunch of things) that with the upcoming release of Vanguard users should be aware there is a not-insignificant chance of ban for using _any_ bot?